$OpenBSD: patch-src_sha1-sse2_S,v 1.3 2012/12/21 17:10:39 landry Exp $
chunk 2 needed on hppa, otherwise fails with
sha1-sse2.S:469: Error: unknown pseudo-op: `.note.gnu'
--- src/sha1-sse2.S.orig	Thu Jan 21 18:01:28 2010
+++ src/sha1-sse2.S	Fri Dec 21 10:02:29 2012
@@ -13,7 +13,7 @@
 #define MANGLE(x) x
 #endif
 
-#if defined(__i386__) || defined(__x86_64__)
+#if (defined(__i386__) || defined(__x86_64__)) && !defined(__PIC__)
 
 .globl  shasse2_init;
 .globl  shasse2_ends;
@@ -466,5 +466,5 @@ cpuid_exit:
 
 
 #ifdef __ELF__
-.section .note.GNU-stack,"",%progbits
+	.section .note.GNU-stack,"",%progbits
 #endif
